osh/builtin.py

```diff
@@ -277,6 +280,28 @@ def _AppendParts(s, spans, max_results, join_next, parts):
   return done, join_next
 
 
+TIMES_SPEC = _Register('times')
+
+class Times(object):
+  def __call__(self, arg_vec):
+    if "SC_CLK_TCK" in os.sysconf_names:
+      clock_ticks_per_second = os.sysconf("SC_CLK_TCK")
+      # Convert resource usage (clock ticks) to seconds before pretty printing
+      print("%dm%1.3fs %dm%1.3fs" %
```

Comment:
Hm I don't know this API / builtin very well, but can it be done with os.times() ?

https://docs.python.org/2/library/os.html#os.times

Reply:
Ya, that looks like that'll work better. I'll test it out